apps/hash-ai-worker-ts/src/main.ts

```diff
@@ -76,6 +77,11 @@ async function run() {
     port: parseInt(getRequiredEnv("HASH_GRAPH_API_PORT"), 10),
   });
 
+  const vaultClient = createVaultClient();
+  if (!vaultClient) {
+    throw new Error("Vault client not created");
+  }
```

Comment:
I think the vault environment variables still need to be added to `docker-compose.ai.yml` for CI to pass, and so that the woker can be run in a docker container locally

Reply:
Added in https://github.com/hashintel/hash/pull/4357/commits/d923735f701c341395fb976994b329abe1be0ac2